cmd/quadlet/main_test.go

```diff
@@ -222,28 +221,39 @@ func TestUnitDirs(t *testing.T) {
 		err = os.RemoveAll(quadlet.UnitDirAdmin)
 		assert.Nil(t, err)
 
-		systemdDir := filepath.Join("/", "systemd")
-		userDir := filepath.Join("/", "users")
-		err = os.Mkdir(systemdDir, 0755)
-		assert.Nil(t, err)
-		err = os.Mkdir(userDir, 0755)
-		assert.Nil(t, err)
-		err = os.Symlink(userDir, filepath.Join(systemdDir, "users"))
-		assert.Nil(t, err)
-		err = os.Symlink(systemdDir, quadlet.UnitDirAdmin)
-		assert.Nil(t, err)
+		createDir := func(path, name string) string {
+			dirName := filepath.Join(path, name)
+			err = os.Mkdir(dirName, 0755)
+			assert.Nil(t, err)
```

Comment:
not blocking as it is pre existing and done in way to many other places but error checks should be done with
`assert.NoError()` because that prints a better message when it fails.

Reply:
Yeah, I just copy pasted the code. I see that it's being fixed in #24974